assets/src/stories-editor/components/editor-carousel/indicator.js

```diff
@@ -29,27 +36,101 @@ const Indicator = ( { pages, currentPage, onClick } ) => {
 	/* translators: %s: Page number */
 	const toolTip = ( pageNumber ) => sprintf( __( 'Go to page %s', 'amp' ), pageNumber );
 
+	const [ draggedPage, setDraggedPage ] = useState( null );
+	const { movePageToPosition, initializePageOrder } = useDispatch( 'amp/story' );
```

Comment:
`movePageToPosition` & co. are used specifically for the reorder view as it allows cancelling the whole reeorder operation to get back the initial order.

In this case, for dragging the carousel indicators, this is not really necessary and there's no way to reset the order anyway.

I'd use `moveBlockToPosition` from the `core/block-editor` store directly and not bother with `movePageToPosition` and `initializePageOrder`

Reply:
Changed in 3d517fc.